sdk/metric/internal/aggregate/sum.go

```diff
@@ -49,7 +47,7 @@ func (s *valueMap[N]) measure(ctx context.Context, value N, fltrAttr attribute.S
 
 	v.attrs = attr
 	v.n += value
-	v.res.Offer(ctx, t, exemplar.NewValue(value), droppedAttr)
+	v.res.Offer(ctx, zeroTime, exemplar.NewValue(value), droppedAttr)
```

Comment:
NewValue is also a constant overhead. Why don't we short-circuit exemplars altogether with `if v.res != nil`?

Reply:
this is now also removed